.github/workflows/conda-package.yml

```diff
@@ -63,7 +63,7 @@ jobs:
 
       - name: Build conda package
         run: |
-          CHANNELS="-c conda-forge -c https://software.repos.intel.com/python/conda --override-channels"
+          CHANNELS="-c conda-forge -c conda-forge/label/python_rc -c https://software.repos.intel.com/python/conda --override-channels"
```

Comment:
should we add some sort of logic so that python_rc channel is only available for 3.14? And won't we have to change it when 3.14 is no longer in RC stage?

thinking about it, if we want to test against RC (I'm not convinced we should, but I don't mind either way) should we just introduce a dedicated workflow for it?

that was it isn't essential/a requirement to pass to merge PRs, and so we can only have to update matrix here, and version there